recipe/patches/0010-abseil-numeric-limits.patch

```diff
@@ -0,0 +1,40 @@
+From 518984e432e0597fd4e66a9c52148e8dec2bb46a Mon Sep 17 00:00:00 2001
+From: Derek Mauro <dmauro@google.com>
+Date: Thu, 8 Sep 2022 08:07:13 -0700
+Subject: [PATCH] Fix stacktrace header includes
+
+Fixes #1275
+
+PiperOrigin-RevId: 472990940
+Change-Id: I1251b01b09e6a9baac52ae4df443714432115e90
+---
+ absl/debugging/internal/stacktrace_riscv-inl.inc | 2 ++
+ absl/debugging/internal/stacktrace_x86-inl.inc   | 2 --
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/3rdparty/chromium/third_party/abseil-cpp/absl/debugging/internal/stacktrace_riscv-inl.inc b/src/3rdparty/chromium/third_party/abseil-cpp/absl/debugging/internal/stacktrace_riscv-inl.inc
+index a9e2c323e7..20183fa321 100644
+--- a/src/3rdparty/chromium/third_party/abseil-cpp/absl/debugging/internal/stacktrace_riscv-inl.inc
++++ b/src/3rdparty/chromium/third_party/abseil-cpp/absl/debugging/internal/stacktrace_riscv-inl.inc
+@@ -30,6 +30,8 @@
+ #include <cassert>
+ #include <cstdint>
+ #include <iostream>
++#include <limits>
++#include <utility>
+ 
+ #include "absl/base/attributes.h"
+ #include "absl/debugging/stacktrace.h"
+diff --git a/src/3rdparty/chromium/third_party/abseil-cpp/absl/debugging/internal/stacktrace_x86-inl.inc b/src/3rdparty/chromium/third_party/abseil-cpp/absl/debugging/internal/stacktrace_x86-inl.inc
+index 1354cb3769..9fbfcf767a 100644
+--- a/src/3rdparty/chromium/third_party/abseil-cpp/absl/debugging/internal/stacktrace_x86-inl.inc
++++ b/src/3rdparty/chromium/third_party/abseil-cpp/absl/debugging/internal/stacktrace_x86-inl.inc
```

Comment:
Ugh, vendored abseil inside vendored chromium... 😑 

Not that I think it's likely, but I checked for previous attempts to package it in staged-recipes, and opened https://github.com/conda-forge/staged-recipes/issues/21431